packages/suite-desktop-ui/src/support/Router.tsx

```diff
@@ -44,7 +45,8 @@ const components: { [key: string]: ComponentType<any> } = {
     'wallet-index': Transactions,
     'wallet-receive': Receive,
     'wallet-details': WalletDetails,
-    'wallet-tokens': WalletTokens,
+    'wallet-tokens-coins': Coins,
+    'wallet-tokens-hidden': HiddenTokens,
```

Comment:
What are the benefits and cons of having separate route for it?

(Not related to this but we have /symbol and account index behind `#` so that it is not leaked but we somehow "leak" it by staking and tokens, that it is network with staking and tokens)

Reply:
Like when we want to push user to specific subpage of tokens. Right now there are just Coins and Hidden but there will be also Collectibles (NFTS) section.